docs/modular-resources/key-concepts.md

```diff
@@ -16,8 +16,15 @@ description: "The key concepts behind how Viam's resource APIs and models are un
 no_list: true
 ---
 
-You can extend the features of an existing {{< glossary_tooltip term_id="resource" text="resource" >}} API by [creating a custom module](/extend/modular-resources/create/) to define a new [model](#models) that implements that API.
-A custom module can provide one or more modular resource models.
+Viam's [Robot Development Kit (RDK)](/internals/rdk/) provides built-in support for a variety of {{< glossary_tooltip term_id="resource" text="resources" >}}:
```

Comment:
I wonder if these few lines are more approachable than what’s on the top level page now but we can address that separately

Reply:
Let's try it!